### DISCUSSION TITLE
Doubts in Task for LambdaTest GitHub Octernship

### DISCUSSION THREAD

**Discussion by Vaibhav-22-dm** [community/community#50287] [GitHub Education] [answered]
Hello Everyone!

First of all, I would like to thank GitHub and LambdaTest for providing an excellent opportunity to all the fellow open source contributors.

I have a few doubts regarding the task of LambdaTest GitHub Octernship. This is the link for the Task instructions: [click here](https://github.com/LambdaTest/Lambdatest-GitHub-Octernship). 

Following are my doubts regarding the task:

- As mentioned in the Task Instructions - 
  ```
  Use a lightweight graph library which can render a time series of 365 days
  ``` 
   but how can we get network logs for 365 days by repeating the process for more than 100 tests. Running 100 tests in a single day will result in getting network logs with timestamp having the same date.

- It is mentioned in the Task Instructions - 
  ```
  Click on each header navigation items
  ``` 
  So, does **header navigation items** mean the _Platform, Enterprise, Resources, Developers_ and _Pricing_ buttons on the website as shown in the image below.
  
  ![image](https://user-images.githubusercontent.com/79047616/225573648-3de0fa2a-4263-4302-a2d9-36cba28089bf.png)
  
  Also, three of these navigation items contain a dropdown, so do we have to write a script to click on each item of the dropdown menu.

- It is mentioned in the Task Expectations - 
  ```
  Making changes on the auto generated feedback branch to complete the task
  ``` 
  while here in the first comment of this PR it is mentioned - 
  ```
  Click the Files changed or Commits tab to see all of the changes pushed to main since the assignment started
  ```
  So, in which branch (main or feedback) am I supposed to push my code?

- Regarding the visualization of logs, are we supposed to show three graphs with y-axis representing:
  ```
  Total count of logs
  Logs count/average stacked by log type (mime-type)
  Logs count/average stacked by log level (Status Codes)
  ```
  respectively in the three graphs and x-axis being 365 days. 

- For the following two tasks: 
  ```
  Logs should be able to filter on the UI by any available key-value
  Logs should be searchable with a keyword using an API to fetch the results
  ```
  Are we supposed to show a list of logs in the form of cards, with a search field at the top and a form to add filters based on available key-value pair?

I request people from GitHub Octernship Team or fellow contributors to please help me out with these doubts.

**Comment by samus-aran** [comment, marked as answer]
Hi @Vaibhav-22-dm 👋 

We would recommend students working on Octernship projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All projects have different preferred methods of communication but here are a few places to check where best to raise inquiry:


- **Octernships Project’s README** - This may include instructions on how and where to raise an inquiry.
- **Octernships Project’s Discussions** - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- **Octernships Project’s Issues** - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍

**Reply by Vaibhav-22-dm** [reply]
Okay @samus-aran . I will try raising an inquiry through your proposed ways. Thanks for answering.

**Comment by Vaibhav-22-dm** [comment]
Yeah Sure.

**Comment by saaxena** [comment]
hi @Vaibhav-22-dm  i am also a part of lambda test octernship and i have few problems while working with the assignment . can i get your email or any messaging platform address so we can connect more . it will be a great help for me .
Thank you !

**Comment by TsarPrince** [comment]
@samus-aran Thankyou for your reply on this, but I cannot find any information about the Lambdatest's project maintainer in README.md, neither it has the Project's discussions enabled. What to do in such scenario, I am really clueless.

**Comment by akgupta0777** [comment]
Wait this guy posted doubt 3 weeks ago and github sent me the mail regarding octernships event on march 28th. Thanks GitHub.
